release-notes.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 3.0.6
+
+### Polish for CHT developent guide
+
+Clean up and polish for: https://docs.communityhealthtoolkit.org/docs/application_development/guides/from_scratch/001_introduction.html
+ - Fixes `initialise-project-layout` for today's best practices
+ - Clean up requirements for passing an instance, see #218
```

Comment:
imo "release notes" are different from a "changelog" (which we had preivously). 

if we have clear descriptive commit messages, there isn't any point in maintaining a changelog. this is just duplicating commit messages (one sentence description, link to issue, etc.). what is the value add here? 

some feedback: [this commit message](https://github.com/medic/medic-conf/commit/5c5e0f0eddc958c36ac383ba66ca0e67ece44057) which should contain the same info you have here in the changelog is not particularly clear. see guidance [here](https://github.com/medic/medic-docs/blob/master/development/workflow.md#commits)

in contrast, release notes provide a forum for deep information about changes between versions. if you agree with this distinction, it would be great to see some comments added here to that effect

Reply:
Yep that all sounds legit :+1: It is a different process from `cht-core`, hence the clarification needed